library/Freckle/App/Test/CommonTests.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Freckle.App.Test.CommonTests
```

Comment:
Another thing I considered adding here was `describeUniverse`....

This would be modeled after our `describeContentArea` function which looks like
```haskell
describeAllContentAreas :: (ContentArea -> SpecWith a) -> SpecM a ()
describeAllContentAreas tests = do
  for_ universe $ \contentArea ->
    describe ("for content area " <> show contentArea) $ tests contentArea
```

Reply:
Thoughts?